src/status_im/contexts/profile/contact/contact_review/view.cljs

```diff
@@ -0,0 +1,45 @@
+(ns status-im.contexts.profile.contact.contact-review.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.contexts.profile.contact.contact-review.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [public-key customization-color]
+         :as   profile}     (rf/sub [:contacts/current-contact])
+        {contact-request-id :id 
+         :as contact-request}     (rf/sub [:activity-center/contact-request-from-contact-id public-key])
+        ;; TODO: remove :blue when #18733 merged.
+        customization-color (or customization-color :blue)
+        full-name           (profile.utils/displayed-name profile)
+        profile-picture     (profile.utils/photo profile)
+        on-contact-accept   (rn/use-callback (fn []
+                                               (rf/dispatch [:hide-bottom-sheet])
+                                               (rf/dispatch [:activity-center.contact-requests/accept
+                                                             contact-request-id]))
+                                             [contact-request-id])
+        on-contact-ignore   (rn/use-callback (fn []
```

Comment:
Here I'm using the `activity-center` subscription and events for accessing the `contact-request` data. It may be useful to refactor this to access contact-requests from a separate event/sub namespace that manages all contact requests directly. But I'm not sure if that's necessary since `activity-center` seems to be doing that.

Reply:
I don't have any strong opinion about it but maybe @ilmotta any thought?

Reply:
We need to be a little bit careful with this piece of data.

Pending contact requests are actually fetched from notifications. Back when I wrote that part I added a long docstring explaining why that works and why it's not the ideal solution. See https://github.com/status-im/status-mobile/blob/62f68ff44b10173946c8bf5919d3209f8836c22d/src/status_im/contexts/shell/activity_center/events.cljs#L411-L418

So just be aware that grabbing a specific contact request from what's stored in the app-db has limitations, but should be fine for now (unless the user has "lots" of pending contact requests).

Reply:
Thank you for providing this context 🙌  

I think this PR assumes the subscribed contact-request is pending, but I can try naming the subscription something like `pending-contact-request-from-contact-id` so the reader knows this limitation, thoughts?

Also do you think I should create a separate issue for possibly rewiring the way contact requests are stored? For example, maybe we can store them in a map that's indexed by contact-id? I suppose that would assume we can only have one contact-request from a person at a time though.

Reply:
Something I'm just noticing is that when we try to supporting dismissing a notification, we'll need to support retrieving dismissed contact-requests. But I think from what you mention, that currently is not possible because the contact request will be marked as read when dismissed, and then removed from the activity-center. 

Am I understanding this correctly?

Reply:
@seanstrom, one thing that could clarify things a bit is that we intentionally descoped managing pending contact requests (see the feature in Figma https://www.figma.com/file/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?type=design&node-id=2010-507764&mode=design&t=to5BwBfcY6wkr3s1-0).

I remember back then I thought a good excuse to improve the client/server code would be if I worked on that feature because it would help me understand the ideal solution. But this doesn't seem to be a priority yet (you could double-check, that was a long time ago). Notifications of type _contact request_ and contact requests themselves are different entities in the backend db, so ideally our code should improve on this area.

> Also do you think I should create a separate issue for possibly rewiring the way contact requests are stored?

Sure! 👍🏼